src/zone/point.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * @class
+ * @description Represents coordinate point
+ */
+export class Point {
```

Comment:
dont see any sense in get/set function here. remove if you have no real goal for it

Reply:
If this class will be used in other modules outside the zone module, for example for describing coordinates of a bullet or tank, get/set funcs will be useful.

Reply:
For current implementation I dont see any reasons to keep them. Please, use YAGNI priciples cause it help make apps more clear and faster https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it

Reply:
I don't get it. I AM using this accessors, because zone is moving and shrinking, I'm calculating it's position and shape, that is described by this coordinates, in addition to some other characteristics. So I store and set this coordinates. Why should I remove this setters if they are used, please, explain this.